[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) ![GitHub stars](https://img.shields.io/github/stars/mumdark/Awesome-Deep-Learning-in-Biology-and-Disease-Papers?color=yellow&label=Stars) ![GitHub forks](https://img.shields.io/github/forks/mumdark/Awesome-Deep-Learning-in-Biology-and-Disease-Papers?color=blue&label=Forks) 

# Awesome-Deep-Learning-in-Biology-and-Disease-Papers <img src="./lattice1.png" align="right" height="200" />

This is a curated collection of papers about ``deep learning applications`` in biology and disease, each providing code for learning or reproducibility purposes.

## Table of Contents

- [Guidelines or reviews for deep learning applications](#guidelines-or-reviews-for-deep-learning-applications)
- [Genome](#genome)
- [Transcriptome](#transcriptome)
- [Epigenome](#epigenome)
- [Proteome](#proteome)
- [Metabolomics](#metabolomics)
- [Microbiome](#microbiome)
- [Multiomics](#multiomics)
- [Radiomics](#radiomics)
- [Benchmarks](#benchmarks)
- [Research group and people](#research-group-and-people)
- [List of Abbreviations](#list-of-abbreviations)
- [Contact](#contact)

## Guidelines or reviews for deep learning applications
1. [2021 Nature Reviews Genetics] **Navigating the pitfalls of applying machine learning in genomics**[[paper]](https://doi.org/10.1038/s41576-021-00434-9)

2. [2022 Nature Reviews Genetics] **Obtaining genetics insights from deep learning via explainable artificial intelligence**[[paper]](https://doi.org/10.1038/s41576-022-00532-2)

3. [2021 Nature Reviews Molecular Cell Biology] **A guide to machine learning for biologists**[[paper]](https://doi.org/10.1038/s41580-021-00407-0)

4. [pytorch-examples] **Simple examples to introduce PyTorch**[[code]](https://github.com/jcjohnson/pytorch-examples)

## Genome
1. [2023 Nature Genetics | <ins style="color: green;"><u> MLP/CNN/ResNet(one-dimensional) </u></ins>]: **Inference of chronic obstructive pulmonary disease with deep learning on raw spirograms identifies new genetic loci and improves risk models**[[paper]](https://doi.org/10.1038/s41588-023-01372-4)[[code]](https://github.com/Google-Health/genomics-research/tree/main/ml-based-copd)

## Transcriptome
1. [2021 Nature Biotechnology | <ins style="color: green;"><u> CNN + GVAE + Gene ranking (GSEA) </u></ins>]: **Prediction of drug efficacy from transcriptional profiles with deep learning**[[paper]](https://doi.org/10.1038/s41587-021-00946-z)[[code]](https://github.com/kekegg/DLEPS)

2. [2023 Nature Biotechnology | <ins style="color: green;"><u>  GNN + Matrix Factorization </u></ins>]: **Supervised discovery of interpretable gene programs from single-cell data**[[paper]](https://doi.org/10.1038/s41587-023-01940-3)[[code]](https://github.com/dpeerlab/spectra/)

## Epigenome
1. 

## Proteome
1.  [2017 PNAS | <ins style="color: green;"><u> CNN + LSTM </u></ins>]: **De novo peptide sequencing by deep learning**[[paper]](https://doi.org/10.1073/pnas.1705691114)[[code]](https://github.com/nh2tran/DeepNovo)
2.  [2022 Briefings in Bioinformatics | <ins style="color: green;"><u> CNN + GAT </u></ins>]: **Learning spatial structures of proteins improves protein–protein interaction prediction**[[paper]](https://doi.org/10.1093/bib/bbab558)[[code]](https://github.com/xzenglab/TAGPPI/tree/main)

## Metabolomics
1.  [2022 Nature Medicine | <ins style="color: green;"><u> MLP + ResNet </u></ins>]: **Metabolomic profiles predict individual multidisease outcomes**[[paper]](https://doi.org/10.1038/s41591-022-01980-3)[[code]](https://github.com/thbuerg/MetabolomicsCommonDiseases)

## Microbiome
1. 

## Multiomics
1.  [2023 Nature Methods | <ins style="color: green;"><u> Stochastic Kinetic Model </u></ins>]: **Dictys: dynamic gene regulatory network dissects developmental continuum with single-cell multiomics**[[paper]](https://doi.org/10.1038/s41592-023-01971-3)[[code]](https://github.com/pinellolab/dictys)
2.  [2024 Genome Biology |  <ins style="color: green;"><u> VAE + GAN + Contrastive Learning </u></ins>]:  **TMO-Net: an explainable pretrained multi-omics model for multi-task learning in oncology**[[paper]](https://doi.org/10.1186/s13059-024-03293-9)[[code]](https://github.com/FengAoWang/TMO-Net)

## Radiomics

## Benchmarks

1.  [2023 Nature Communications]: **Systematic comparison of tools used for m6A mapping from nanopore direct RNA sequencing**[[paper]](https://doi.org/10.1038/s41467-023-37596-5)[[code]](https://github.com/zhongzhd/ont_m6a_detection)

2.  [2023 Genome Biology]: **Comprehensive benchmark and architectural analysis of deep learning models for nanopore sequencing basecalling**[[paper]](https://doi.org/10.1186/s13059-023-02903-2)[[code]](https://github.com/marcpaga/basecalling_architectures)


## Research group and people
1.[Singapore] **Jonathan Göke**：We’re a computational genomics group searching for answers to these questions. We develop algorithms and machine learning methods and investigate genomics and transcriptomics data to better understand human diseases such as cancer. Our group specifically works on third generation long read RNA sequencing technology. We are located at the Genome Institute of Singapore. Read more about us, our research and publications, or available scholarships (PhD, internships) and positions. You can find more information about methods from the team on our github page [[home]](https://jglab.org/)

## List of Abbreviations
- MLP: Multilayer Perceptron
- ResNet: Residual neural network
- AE: Autoencoder
- VAE: Variational Autoencoder
- GVAE: Graph Variational Autoencoder
- CNN: Convolutional Neural Network
- GAT: Graph Attention Network
- GAN: Generative Adversarial Network

## Contact

<a href="mailto:ningyan1212@gmail.com">Email：mumdark &nbsp;酩酊</a>